Feature/modal

NurzhanovBekzat opened the pull request.

Kamchybekk committed:
Update README.md

ZhakshylykNasip committed:
Merge pull request #1 from peaksoft-school/development

Development

Commit:
created modal

Commit:
added modal

ZhypargulKanybekova commented on package.json:
```diff
@@ -3,12 +3,16 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "styled-components": "^6.0.3",
```
style component неге кочурдун

Commit:
deleted styled-components

Commit:
fixed conflicts

AziatAbd commented on src/components/UI/modal/Modal.jsx:
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+export const ModalUi = ({ children, ...rest }) => {
+   const [open, setOpen] = React.useState(false)
+   const handleClose = () => setOpen(false)
```
состояние и функция не нужны

AziatAbd commented on src/components/UI/modal/Modal.jsx:
```diff
@@ -0,0 +1,38 @@
+
+   return (
+      <div>
+         <Backdrop onClose={handleClose} open={open}>
```
пусть onClose и open через props придут

AziatAbd commented on src/components/UI/modal/Modal.jsx:
```diff
@@ -0,0 +1,38 @@
+   const handleClose = () => setOpen(false)
+
+   return (
+      <div>
```
delete div

Commit:
finished modal task

AziatAbd commented on src/App.js:
```diff
@@ -1,4 +1,18 @@
 function App() {
-   return <div>Task Trecker js-9</div>
+   return (
+      <div>
+         <Input
+            placeholder="Add a description"
+            type="text"
+            padding="8px 16px"
+            height="5.18rem"
+            width="42.68rem"
+            borderRadiuss="8px"
+            border="1px solid #D0D0D0;"
+         />
+      </div>
```
зачем это?

AziatAbd commented on src/components/UI/modal/Modal.jsx:
```diff
@@ -0,0 +1,36 @@
+import { Box, Modal, styled } from '@mui/material'
+import React from 'react'
+
+export const ModalUi = ({ children, ...rest }) => {
```
где open и onCLose?

Commit:
import { TextField, styled } from '@mui/material'
import React, { forwardRef } from 'react'

export const Input = forwardRef(
   ({ type, borderRadius, placeholder, ...props }, ref) => {
      return (
         <StyledInput
            type={type}
            ref={ref}
            placeholder={placeholder}
            {...props}
         />
      )
   }
)

const StyledInput = styled(TextField)((props) => ({
   width: props.width,
   marginLeft: '25%',
   marginTop: '100px',
   border: props.border,
   '& .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input': {
      height: props.height,
      padding: props.padding,
   },
   '& .css-1qi90xi-MuiInputBase-root-MuiOutlinedI

Commit:
code changes

Commit:
feature modal changed

AziatAbd committed:
fix bugs

AziatAbd committed:
fix conflicts
